Tests/KDTreeTests/KDTreeTests.swift

```diff
@@ -192,12 +192,18 @@ class KDTreeTests: XCTestCase {
             }
         }
     }
+    
+    func test03_Build() {
+        let points = Array(0..<100000).map({_ in CGPoint(x: CGFloat.random(), y: CGFloat.random())})
+        let _ = KDTree(values: points)
```

Comment:
👍 for adding a test, but maybe we should add some sort of Expectation to make it whole.

Like `XCTAssertEqual(KDTree(values: points).count, 100000)` you know?

Also keep in mind that this is only the test file for tests on Linux, on iOS/macOS there are all the other tests being run you can find in the `Tests` subfolder.
I'm sorry, I know I promised to merge the files for the Linux tests with the macOS tests eventually, will go to my To-Do-list for next weekend.